### PR TITLE
Docs todos

### PR DESCRIPTION
**Pull request by registerrier**
**Description**

This pull request proposes a reorganization of the tutorial.rst page to distribute tutorials in sections:
- getting started
- Core tutorials
- Advanced tutorials

This comes from suggestions made at the user test to better organize tutorials depending on the expected knowledge of users and to improve visibility of the left column.  
**Dear reviewer**

this is a proposal for the overall structure. The exact location of each tutorial can be later changed.

Opinions @facero , @adonath , @AtreyeeS @QRemy ?

### CODE REVIEW

File: docs/tutorials/index.rst [hunk 1]
```diff
@@ -21,88 +21,125 @@ handbook`_ and the `Astropy Hands-On Tutorial`_.
...
+*Image analysis*
+
+- `Source detection <../notebooks/detect.html>`__ | *detect.ipynb*
+
+*Multi-instrument analysis*
+
+- `Multi instrument joint 3D and 1D analysis <../notebooks/analysis_mwl.html>`__ | *analysis_mwl.ipynb*
+- `A Fermi-LAT analysis with Gammapy <../notebooks/fermi_lat.html>`__ | *fermi_lat.ipynb*
+
+*Sensitivity estimation and observation simulation*
```
Comment (AtreyeeS) [inline docs/tutorials/index.rst]: There is already a Simulations, Sensitivity, Observability section above, so this section might add some confusion -  which notebook to see for sensitivity analysis?

File: docs/tutorials/index.rst [hunk 2]
```diff
@@ -21,88 +21,125 @@ handbook`_ and the `Astropy Hands-On Tutorial`_.
...
+   ../notebooks/spectrum_simulation.ipynb
+   ../notebooks/modeling_2D.ipynb
+   ../notebooks/ring_background.ipynb
+   ../notebooks/overview.ipynb
+   ../notebooks/maps.ipynb
+   ../notebooks/modeling.ipynb
+   ../notebooks/models.ipynb
+   ../notebooks/catalog.ipynb
+
+*Image analysis*
```
Comment (AtreyeeS) [inline docs/tutorials/index.rst]: Is this separate section needed or this can also go under 2-d image analysis?

File: docs/tutorials/index.rst [hunk 3]
```diff
@@ -21,88 +21,125 @@ handbook`_ and the `Astropy Hands-On Tutorial`_.
...
+Advanced tutorials
+------------------
+
+.. toctree::
+   :hidden:
+
+   ../notebooks/analysis_mwl.ipynb
+   ../notebooks/detect.ipynb
+   ../notebooks/cta_sensitivity.ipynb
+   ../notebooks/spectrum_simulation.ipynb
```
Comment (AtreyeeS) [inline docs/tutorials/index.rst]: spectrum_simulations need not come again

File: docs/tutorials/index.rst [hunk 4]
```diff
@@ -21,88 +21,125 @@ handbook`_ and the `Astropy Hands-On Tutorial`_.
...
 
+- `Overview <../notebooks/overview.html>`__  | *overview.ipynb*
 - `CTA with Gammapy <../notebooks/cta.html>`__ | *cta.ipynb*
 - `H.E.S.S. with Gammapy <../notebooks/hess.html>`__ |  *hess.ipynb*
-- `Fermi-LAT with Gammapy <../notebooks/fermi_lat.html>`__ | *fermi_lat.ipynb*
 
-**Analyses**
-
-*3-dim sky cube analysis*
+*1D spectral analysis*
```
Comment (AtreyeeS) [inline docs/tutorials/index.rst]: Nitpicking, but 1-dim like in the other two

File: docs/tutorials/index.rst [hunk 5]
```diff
@@ -21,88 +21,125 @@ handbook`_ and the `Astropy Hands-On Tutorial`_.
...
-**Simulations, Sensitivity, Observability**
+*Simulations, Sensitivity, Observability*
 
-- `3D map simulation <../notebooks/simulate_3d.html>`__ | *simulate_3d.ipynb*
 - `1D spectrum simulation <../notebooks/spectrum_simulation.html>`__ | *spectrum_simulation.ipynb*
+- `3D map simulation <../notebooks/simulate_3d.html>`__ | *simulate_3d.ipynb*
+
+Advanced tutorials
+------------------
+
```
Comment (AtreyeeS) [inline docs/tutorials/index.rst]: Maybe add a line here why these are "advanced tutorials" - they don't demonstrate a complete data reduction and analysis like in the previous section, but rather handling of specific gammapy-objects like maps or models?

File: docs/tutorials/index.rst (hunk 2 above)
Comment (registerrier) [reply docs/tutorials/index.rst]: Added specific info on detection

File: docs/tutorials/index.rst (hunk 1 above)
Comment (registerrier) [reply docs/tutorials/index.rst]: remove simulation

File: docs/tutorials/index.rst (hunk 4 above)
Comment (registerrier) [reply docs/tutorials/index.rst]: done

File: docs/tutorials/index.rst (hunk 5 above)
Comment (registerrier) [reply docs/tutorials/index.rst]: Done

**Comment by registerrier** [comment]
unrelated fail. Merging

**Pull request merged**